pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Artsy Product Velocity Team <product-velocity@artsymail.com>"]
 [tool.poetry.dependencies]
 boto3 = "^1.26.61"
 Jinja2 = "^3.1.2"
-python = "^3.9.10"
+python = ">=3.9,<3.11"
```

Comment:
@artsyjian Degas is relaxed about python versions because it's packaged as a lib to be used by other repos (Zenith jobs use the Degas utils). If this is an application rather than a library, then pinning is fine (and we mainly pin in our apps for stability reasons).

Reply:
good to know. thanks.